2996_reflection/reflection.md

```diff
@@ -7163,16 +7163,24 @@ consteval access_context via(info cls) const;
 consteval bool is_accessible(info r, access_context ctx);
 ```
 
-[#]{.pnum} *Constant When*: `r` does not represent a member or unnamed bit-field of an incomplete class.
+[#]{.pnum} *Constant When*: `r` does not represent
 
-[#]{.pnum} Let `$P$` be a program point defined as follows:
+* [#.#]{.pnum} a member or unnamed bit-field of an incomplete class or
+* [#.#]{.pnum} a direct base class relationship between a base class and an incomplete derived class.
 
-* [#.#]{.pnum} If `ctx.scope()` represents the null reflection, then [FIXME].
-* [#.#]{.pnum} Otherwise, [FIXME].
+[#]{.pnum} Let `$S$` be a scope ([basic.scope.scope]) defined as follows:
+
+* [#.#]{.pnum} If `ctx.scope()` represents the null reflection, then the global scope.
+* [#.#]{.pnum} Otherwise, if `r` represents a function `$F$`, then the block scope introduced by the `$compound-statement$` of the `$function-body$` in the definition of `$F$`.
+* [#.#]{.pnum} Otherwise, if `r` represents a class `$C$`, then the class scope of `$C$`.
+* [#.#]{.pnum} Otherwise, if `r` represents a namespace `$N$`, then the namespace scope of `$N$`.
+
+[#]{.pnum} Let `$P$` be an unspecified point enclosed by `$S$` such that no function parameter scope, class scope, or namespace scope intervenes between `$P$` and `$S$`.
```

Comment:
I think this would be better suited by pointing to `current().scope()`. 

Like just:
> If `ctx.scope()` does not represent the null reflection, then let `$P$` be a program point such that `access_context::current().scope() == ctx.scope()` is `true` when invoked at `$P$`.

This avoids reduplicating the whole definition of `current().scope()`, and plus this is exactly the intent of the wording.